runtimes/raku.nix

```diff
@@ -0,0 +1,58 @@
+{pkgs, piston, ...}:
+let
+    pkg = pkgs.rakudo;
+in piston.mkRuntime {
+    language = "raku";
+    version = pkg.version;
+
+    aliases = [
+        "raku"
```

Comment:
this doesn't need to be an alias

Reply:
good catch. I just copied from the old pkg, but I'll remove it here